modules/client/test/integration/client/encoding.test.ts

```diff
@@ -122,45 +118,27 @@ describe("Client", () => {
           permission: Permissions.EXECUTE_PERMISSION,
         },
       ];
-      let actions: DaoAction[] = [];
+      let actions: IEncodingResult[] = [];
       for (let i = 0; i < paramsArray.length; i++) {
         const params = paramsArray[i];
         actions.push(client.encoding.freezeAction(daoAddresses[i], params));
       }
-      const decoder = new TextDecoder();
+
       for (let i = 0; i < actions.length; i++) {
         const action = actions[i];
         expect(typeof action).toBe("object");
         expect(action.to).toBe(daoAddresses[i]);
         expect(action.data).toBeInstanceOf(Uint8Array);
       }
-      expect(
-        decoder.decode(actions[0].data) === decoder.decode(actions[1].data),
-      ).toBe(false);
+      expect(actions[0].data === actions[1].data).toBe(false);
     });
     it("Should encode an update metadata raw action", async () => {
       const context = new Context(contextParamsLocalChain);
       const client = new Client(context);
 
-      const params: IMetadata = {
-        name: "New Name",
-        description: "New description",
-        avatar: "https://theavatar.com/image.jpg",
-        links: [
-          {
-            url: "https://discord.com/...",
-            name: "Discord",
-          },
-          {
-            url: "https://twitter.com/...",
-            name: "Twitter",
-          },
-        ],
-      };
-
       const installEntry = await client.encoding.updateMetadataAction(
```

Comment:
Same

Reply:
a mistake from where I started to refactor some stuff.
It will be added back